xmtp_api_grpc/src/grpc_api_helper.rs

```diff
@@ -73,10 +89,12 @@ impl Client {
             let uri =
                 Uri::from_str(&host).map_err(|e| Error::new(ErrorKind::SetupError).with(e))?;
 
-            let tls_client = MessageApiClient::with_origin(tls_conn, uri);
+            let tls_client = MessageApiClient::with_origin(tls_conn.clone(), uri.clone());
+            let mls_client = ProtoMlsApiClient::with_origin(tls_conn, uri);
 
             Ok(Self {
                 client: InnerApiClient::Tls(tls_client),
+                mls_client: InnerMlsClient::Tls(mls_client),
```

Comment:
Is there any consumer that would need both of these clients? Wondering if we should just return one or the other based on a flag? Or do we rely on endpoints from both of these clients?

Reply:
`xmtp_mls` is going to need both. My intention is to rely on all of our existing `Query`, `BatchQuery` and `Stream` methods for reading messages from topics, since we don't need to do anything special on the read side.

Reply:
They are implemented as separate GRPC services on the node with separate protobuf services, so they need separate underlying clients (although the clients can share a single connection).